test/src/test/java/org/corfudb/runtime/object/transactions/AbstractTransactionsTest.java

```diff
@@ -48,9 +57,8 @@ protected void OptimisticTXBegin() {
     /**
      * Utility method to start a snapshot TX
      */
-    protected void SnapshotTXBegin() {
-        // By default, begin a snapshot at address 2L
-        Token t2 = new Token(0l, 2l);
+    protected void SnapshotTXBeginWithTimestamp(long snapshotTimestamp) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name 'SnapshotTXBeginWithTimestamp' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651450)